src/components/ListView.jsx

```diff
@@ -1,12 +1,26 @@
 import React, { useEffect, useState } from 'react';
 
-import { collection, onSnapshot } from 'firebase/firestore';
+import { collection, onSnapshot, setDoc, doc } from 'firebase/firestore';
 import { db } from '../lib/firebase';
 import { useNavigate } from 'react-router-dom';
 
+//we need an advice how we can pass a field name as a varible
+//so we can reuse this function to update any field in fireBase
+
```

Comment:
This is an interesting question. I think we can create a custom object in our Firebase config that would allow us to do that, if I'm understanding this correctly. Curious to know more...

Reply:
Sorry for the confusion DJ, we forgot to delete this comment. Eddie helped us out during out Wednesday meeting and this function works as expected now. 
I like your idea to _create a custom object in our Firebase config_ cause I have no idea what does it mean - will be glad learn more how it could be done!